NOTICE

```diff
@@ -0,0 +1,12 @@
+NOTICES
+
+This repository incorporates material as listed below or described in the code.
+
+The initial concepts and development of some of the framework's plugin common modules under pkg/common/plugin are derived from, or
+inspired by the project Container Network Interface - networking for Linux containers (https://github.com/containernetworking/cni).
+It is licensed under the Apache License, Version 2.0.
+
+The loading of framework's configuration from a home directory is derived from or inspired by the project Docker CLI (https://github.com/docker/cli)
```

Comment:
What does this mean? It seems not related to this repository?

Reply:
This is the same as the current notice under `ratify` repo, so I just keep it.